src/main/java/org/waarp/openr66/dao/database/h2/H2TransferDAO.java

```diff
@@ -0,0 +1,36 @@
+package org.waarp.openr66.dao.database;
```

Comment:
Wrong package (missing h2) and therefore need `import org.waarp.openr66.dao.database.DBTransferDAO;`

Same for other XxxTransferDAO